docs/efun/calls/call_out.md

```diff
@@ -5,23 +5,41 @@ title: calls / call_out
 
 ### NAME
 
-    call_out() - delayed function call in same object
+    call_out - delayed function call in same object
 
 ### SYNOPSIS
 
-    void call_out( string | function fun, int delay, mixed arg );
+    int call_out( string | function fun, int | float delay, mixed arg ... );
 
 ### DESCRIPTION
 
-    Set  up  a  call of function <fun> in this_object(). The call will take
-    place in <delay> seconds, with the argument <arg> provided.  <arg>  can
-    be of any type.
+    Schedule a future call of function <fun> in this_object(). The call will
+    take place in <delay> seconds, with each of the arguments <arg> provided.
+    <arg> can be of any type.
 
-    Please  note  that  you  can't  rely on write() or say() in <fun> since
+    If the gametick in the runtime config is less than 1000, you may specify
+    a <delay> as a float in milliseconds (gametick / 1000) representing a 
+    value that is divisible by the gametick.
+
+    For example, if your gametick is set to 250ms in the runtime config,
+    you may perform callouts with a granularity of a quarter of a second.
+    call_out( "function", 0.75 ) will execute the <fun> in 3 game ticks
+    (750 milliseconds), thereby affording more precision over timing
+    when calling out.
+
+    A <delay> of 0 (or 0.0), will perform <fun> on the next game tick, a
+    distance of time defined in the runtime config.
```

Comment:
this is actually not true,  schedule an call_out with delay of 0  will be executed on the same gametick after all the previous call_outs has been executed. 

you need to mention in runtime config there is a `"call_out(0) nest level` setting which limits  how many call_out(0) can be scheduled on an single tick

Reply:
I've submitted a new commit: https://github.com/fluffos/fluffos/pull/713/commits/94575ed6917b1439bbea29bef914d74bcde6499b